biothings/web/query/formatter.py

```diff
@@ -162,10 +162,21 @@ def transform(self, response, **options):
             native: bool, if the returned result is in python primitive types.
             version: bool, if _version field is kept.
             score: bool, if _score field is kept.
-
+            with_total: bool, if True, the response will include max_total documents,
+                and a message to tell how many query terms return greater than the max_size of hits.
+                The default is False.
+                An example when with_total is True:
+                {
+                    'max_total': 100,
+                    'msg': '12 query terms return > 1000 hits, using from=1000 to retrieve the remaining hits',
+                    'hits': [...]
+                }
         """
         options = dotdict(options)
         if isinstance(response, list):
+            max_total = 0
+            count_by_queries = {}
+
```

Comment:
should be conditional based on `options.with_total` value.

Reply:
please read my previous comment: https://github.com/newgene/biothings.api/pull/52#discussion_r934684022